recipe/build.sh

```diff
@@ -5,12 +5,27 @@
 set -vexu -o pipefail
 
 pushd nim
+
 # inject compilers
-echo "gcc.exe = \"x86_64-conda_cos6-linux-gnu-gcc\"" >> config/nim.cfg
-echo "gcc.cpp.exe = \"x86_64-conda_cos6-linux-gnu-gcc\"" >> config/nim.cfg
-echo "gcc.linkerexe = \"x86_64-conda_cos6-linux-gnu-gcc\"" >> config/nim.cfg
-echo "clang.exe = \"x86_64-conda_cos6-linux-gnu-gcc\"" >> config/nim.cfg
-echo "clang.linkerexe = \"x86_64-conda_cos6-linux-gnu-gcc\"" >> config/nim.cfg
+cc=clang
+ldflags="${LDFLAGS}"
+if [[ ${target_platform} =~ linux.* ]]; then
+    cc=gcc
```

Comment:
I think this needs to be equivalent to
```
$PREFIX/bin/x86_64-conda_cos6-linux-gnu-cc
```
since `gcc` is not in `$PATH` on linux. I've posted the linux error as a comment.